src/Request/BasicResolver.php

```diff
@@ -12,7 +12,11 @@ class BasicResolver implements ResolverInterface
     public function resolve()
     {
         if (!isset($_SERVER['REQUEST_METHOD'])) {
-            return new NullRequest();
+            if ((PHP_SAPI == 'cli') && (isset($_SERVER['argv']))) {
```

Comment:
The extra brackets are not needed, and `===` should be used for the string comparison.